orb/orb.yml

```
@@ -300,7 +300,7 @@ commands:
             export DEPLOY_FILE=<< parameters.deploy-file >>
             export APP_NAME=<< parameters.app-name >>
             export SECRET_FILE=<< parameters.secret-file >>
-            export SECRETS=$(yq read $SECRET_FILE -j)
+            export SECRETS=$(yq -o=json $SECRET_FILE)
```

Shouldn't this be like this?
```suggestion
            export SECRETS=$(yq eval -o json $SECRET_FILE)
```

---

Why do you want the eval command ?